doc/sphinx-guides/source/admin/solr-search-index.rst

```diff
@@ -22,7 +22,7 @@ Get a list of all database objects that are missing in Solr, and Solr documents
 
 ``curl http://localhost:8080/api/admin/index/status``
 
-Remove all Solr documents that are orphaned (ie not associated with objects in the database):
+Remove all Solr documents that are detached (ie not associated with objects in the database):
```

Comment:
Is there any particular reason we're changing this from "orphaned" to "detached"? The API endpoint is called "clear-orphans".

Reply:
addressed in https://github.com/IQSS/dataverse/pull/7283/commits/f0a0d0b9f31a8315a5043cc3f9f0a7b872af64a9